my_locustfiles/fund_store.py

```diff
@@ -19,21 +20,42 @@ def list_all_funds(self):
     @task
     def get_a_fund_search(self):
         """
-        Performance test for GET /funds?search_items={fund_name}
+        Performance test for GET /funds?search_items={fund_id}
          that expects a 200.
         """
         with self.client.get(
-            f"/funds?search_items={self.fund_name}",
+            f"/funds?search_items={self.fund_id}",
```

Comment:
Ah also search_items should be a comma separated list of strings like "breakfast,fund" so perhaps create a new variable search_query="breakfast,fund" and use that her instead of fund_id

Reply:
@harryyo Have made it a separated list of strings via a new variable.